### DISCUSSION TITLE
TabletMode

### DISCUSSION THREAD

**Discussion by ROOMBON** [valinet/ExplorerPatcher#512] [Ideas]
Is there a known way to restore tablet mode like in Windows 10?

**Comment by valinet** [comment]
I personally do not know. Also, I don't have much experience with it, if any, at all.

**Comment by ROOMBON** [comment]
I have a Windows tablet and it is very significant for me.
It's just an automatic magnification of each window to the entire screen so you can not see more than one window. And that sliding from top to bottom will close the specific window that is open.
By the way, in Windows 11 they also changed that sliding to the left of the center opens the widgets instead of a "task view" and it would be nice if it could be fixed as well.
Thanks.

**Comment by robmausser** [comment]
Unbelievable that Microsoft would remove Tablet Mode view in Windows 11. What were they thinking? Not like tablets are the hottest selling market in computers right now or anything.... I mean, aren't they making and marketing Surface devices? idiots.

**Comment by XuanChen1234** [comment]
You can ask from @Amrsatrio

**Reply by Amrsatrio** [reply]
Dude, I am not Microsoft or a god. I cannot implement EVERYTHING you've wanted me to, or at least expected me to be able to. Please stop.

**Comment by XiaoTDM** [comment]
Basic Implementation Approachs
1. Restore Application Control Mode: Always run in maximized mode with unique control gestures.
2. Restore Taskbar: Add an extra back button, which can be implemented by mapping it to Windows Key + Backspace shortcut.
3. Restore "Start" Behavior: Replace the desktop so that when closing an application, it returns to "Start" instead of the desktop.
